### DISCUSSION TITLE
Shortcut for adjusting brush size

### DISCUSSION THREAD

**Discussion by pistachio-diffuser** [Orama-Interactive/Pixelorama#776] [Ideas]
Please, add a keyboard shortcut for adjusting a brush size like in blender or other programs. I want to speed up my workflow in Pixelorama. I would love to see this feature. It will improve my experience of using this already wonderful program :)

**Comment by OverloadedOrama** [comment]
I implemented this feature in commit 8732d366bff99e01dd11ccf4f7938854cb131a92 and it will be available in version 0.11. The shortcut is Control + Mouse wheel.

But, because I could not figure out if there is a way to combine a keyboard shortcut with mouse in Godot 3.5, the shortcut is hardcoded and unfortunately cannot be edited in the shortcut settings. However, Godot 4.x seems to support input actions mouse buttons with keyboard modifiers (Control, Shift, Alt etc), so when Pixelorama gets ported to Godot 4.x, the shortcut should be able to become editable.

**Reply by pistachio-diffuser** [reply]
Thank you a lot! You make me cry.
I'm so grateful for your work! I will be waiting for it!

For now, I have created autohotkey script as a temporary solution :)
